week4_Isotopes.Rmd

```diff
@@ -0,0 +1,117 @@
+---
+title: "Isotopes Fractionation"
+subtitle: "Isoptopes are isoDOPE"
+author: "Kevin Rozmiarek"
+date: "Last knitted `r format(Sys.Date(), '%d %b %Y')`"
+output:
+  html_document: 
+    df_print: paged
+    number_sections: yes
+    toc: yes
+    toc_float: true
+    toc_depth: 3
+    code_folding: show
+editor_options:
+  chunk_output_type: inline
+---
+
+# Learning Goals
+
+1. Understand the role of **fractionation factors** in separating isotopes in different reservoirs
+
+2. Understand the difference between **equilibrium isotope fractionation** and **kinetic isotopic fractionation**
+
+3. Be able to numerically demonstrate the isotopic evolution of a water reservoir as a consequence of evaporation
+
+4. Be able to manipulate and deal with bad input data.
+
+# Prerequisites
+
+1. Have an understanding of what **isotopes** are and how to express them in different notations.
+
+2. Basic understanding of **R** programming, flow control, graphing, and tibble manipulation.
+
+# Background
+
+Because we just finished our lesson on what isotopes are and their notation *cough*, it is time to talk about **isotope fractionation**. **Isotope fractionation** is the separation or partitioning of a reservoir(s) stable isotopes due to a processe that use masses differently. Some processes depend on the relative differences in the masses of different isotopes in chemical equilibrium (**equilibrium isotopic fractionation**). Other processes depend on rate-dependent transfers or site-specific chemical binding effects (**kinetic isotopic fractionation**).
+
+Today we are going to talk about the evaporation of water which happens to be an example of a rate-dependent kinetic fractionation. Well... its kinetic in very dry air. Let's not go there for now. The fractionation from a process is described by using an often empirically derived non-dimensional parameter called a **fractionation factor**. Evaporation is considered to be a type of fractionation called a **Rayleigh distillation**. In an **open system**, isotopes will not return to the original reservoir once evaporated. At this point, we can write out a general form for a Rayleigh distillation that will tell us the isotopic ratio of the reservoir as a function of the remaining amount that hasn't been evaporated.
```

Comment:
You mention in the paragraph above this one that the fractionation relates to mass differences and (preferential mass "use") but I think dropping here that in addition to isotopes not "returning" to the reservoir the isotopic make up of the reservoir is "mass-determined" because of this would be a good segway into introducing 18O and 16O and the equations below.... just a thought.